pkgs/by-name/sl/sleek-grub-theme/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, withBanner ? "Grub Bootloader" # use override to specify your own banner text
+, style ? "white" # use override to specify one of "dark" / "orange" / "bigSur"
+}:
+stdenv.mkDerivation {
+  pname = "sleek-grub-theme";
+  version = "unstable-2022-06-04";
```

Comment:
The [unstable manual](https://github.com/NixOS/nixpkgs/blob/af5f850f1aa54587dccac8a8abf814dd9f58059c/pkgs/README.md?plain=1#L334-L336) changes about this: you should prepend the latest version.

However, it has never released before, so there is no latest version.

Reply:
So, what can I do with that... ?

Reply:
I do not know 🙃.  Leave it unchanged or use something like `0.0.0-unstable-2022-06-04`.

Reply:
OK,  then I'll just leave it unchanged,  unless there is more specific guide.